src/cljs/athens/keybindings.cljs

```diff
@@ -101,10 +102,20 @@
       (= type :slash) (cond
                         (= :up direction) (do
                                             (.. e preventDefault)
-                                            (swap! state update :search/index dec))
+                                            (swap! state update :search/index #(dec (if (zero? %) (count slash-options) %)))
+                                            (let [cur-index (:search/index @state)
+                                                  container-el (.. e -target -parentNode -parentNode -nextSibling -firstChild)
```

Comment:
Chaining DOM navigation feels a little fragile in case the structure changes. It might make sense to give the `slash-menu-el` its own HTML id here.

Reply:
Gave the buttons' container and id (`slash-command-container`).